extension/content.js

```diff
@@ -51,6 +52,24 @@ function addReleasesTab() {
 	}
 }
 
+function addBlameParentLinks() {
+	let pathname = location.pathname;
+
+	$('.blame-sha').each(function(index, commitLink) {
+		let $commitLink = $(commitLink)
+		let $blameParentLink = $commitLink.clone();
+		let commitSha = commitLink.href.match(/(?<=commit\/)\w+/)[0];
```

Comment:
@paulmolluzzo That's my point. Because I'm cloning it from the DOM instead of manually constructing it, I don't trust it to not change. E.g. GitHub may decide to implement my first point below:
> when you click the normal commit, to actually go to the exact same line of the committed file where this line was introduced
and then the regex would need to adapt to this change.
I may be wrong to think in this (overprotective future-proof) way, but I hope I've explained my way of thinking.

Using `\w{40}$` is protective in a different direction - with this you say any other link should not match. Which is defensive and could be useful especially for validating input, but I don't see it as very useful when extracting data from something I *expect* to change.